diff --git a/apps/desktop/electron-builder.ts b/apps/desktop/electron-builder.ts
index 9bcd771..10feaa7 100644
--- a/apps/desktop/electron-builder.ts
+++ b/apps/desktop/electron-builder.ts
@@ -114,10 +114,6 @@ const config: Omit<Writable<Configuration>, "electronFuses"> & {
     },
     files: [
         "package.json",
-        {
-            from: ".hak/hakModules",
-            to: "node_modules",
-        },
         "lib/**",
     ],
     extraResources: ["build/icon.*", "webapp.asar"],
